include/odp/api/spec/event.h

```diff
@@ -143,6 +143,20 @@ void *odp_event_user_area(odp_event_t event);
  */
 void *odp_event_user_area_and_flag(odp_event_t event, int *flag);
 
+/**
+ * Set event user flag
+ *
+ * Set (or clear) the user flag.
+ *
+ * The user flag maps to the user flag value of underlying event type (e.g.
+ * odp_packet_user_flag() for packets). If the event does not have user flag,
+ * nothing is done.
```

Comment:
@MatiasElo I prefer to have capability flag or return -ENOSUP kind standard return code to express if implementation does not support feature.

Reply:
Okay, we can discuss about this in tomorrow's call.

Reply:
>Did you mean to have capabilities for those?

Yes